src/core/api/public_api.ts

```diff
@@ -751,6 +765,9 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     /** Emit playback events. */
     let playback$ : ConnectableObservable<IInitEvent>;
 
+    this._priv_speed$.next(this._priv_speedChoices.normal);
```

Comment:
I thought we would be re-initializing it here to avoid impacting the previous content: